#goal definition
string SoundName
bool   SoundLoop
---
#result definition
string Result
---
#feedback
string Feedback
